### ISSUE TITLE
Accessibility: Cannot select these elements on focus on PLP

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#6568]
**Environment Details**
Spartacus: epic/accessibility branch
Browser: Chrome & firefox & Safari
Device: mac
Happens on: http://dev-com-3.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD

**Steps to Reproduce**
- Go to Product List Page

We cannot actually select these by pressing ENTER or SPACEBAR key:
- Minimize Stores, Price, Brands etc
<img width="1239" alt="Screen Shot 2020-02-14 at 11 08 42 AM" src="https://user-images.githubusercontent.com/45409515/74550837-10586e00-4f20-11ea-888b-89022879e99d.png">

- Show More
<img width="1269" alt="Screen Shot 2020-02-14 at 11 09 11 AM" src="https://user-images.githubusercontent.com/45409515/74550882-2c5c0f80-4f20-11ea-9f8c-f79b4a91ac31.png">

- Go to a different page
<img width="1187" alt="Screen Shot 2020-02-14 at 11 10 06 AM" src="https://user-images.githubusercontent.com/45409515/74550957-4bf33800-4f20-11ea-9086-3ec91d845dc7.png">

- Go to grid view
<img width="735" alt="Screen Shot 2020-02-14 at 11 10 29 AM" src="https://user-images.githubusercontent.com/45409515/74550971-57466380-4f20-11ea-90b1-5274248c8265.png">

**Comment by KateChuen** [comment]
Pagination also does not work on StoreFinder Page. Cannot actually select a page, go next or previous, using the ENTER key.

**Comment by kacperknapik** [comment]
This will be fixed in the near facets' refactor. Not a11y related
